static/js/components/ShowClassification.jsx

```diff
@@ -99,6 +112,23 @@ const ClassificationRow = ({ classifications }) => {
 
   const classification = classifications[0];
 
+  const confirmed_classes = ["GO GRANDMA", "GO GRANDMA (HIGH PRIORITY)"];
+  const rejected_classes = ["STOP GRANDMA"];
```

Comment:
We should not be hard coding anything with GRANDMA here. You can add these via an environment variable in the config OR have these defined in a table.